README.md

```diff
@@ -1 +1,42 @@
-# export-service-go
\ No newline at end of file
+# export-service-go
+
+## Starting the service
+You can start the database and minio using `podman-compose up db s3`. This will expose minio on `localhost:9099` 
```

Comment:
It might be helpful to have a **Dependencies** Section above "Starting the service" for things like **podman-compose** and the **minio console** that you reference later in the README.

Reply:
The minio console is started with `podman-compose` and port-forwarded so you can access it locally on `localhost:9099`, so it shouldn't require any additional dependencies. I added a section for the other dependencies though. Thanks, Alec!